client/src/app/management/components/committee-edit/committee-edit.component.html

```diff
@@ -68,8 +68,9 @@ <h2 *ngIf="!isCreateView">{{ editCommitteeLabel | translate }}</h2>
                     formControlName="forward_to_committee_ids"
                     [multiple]="true"
                     [repo]="committeeRepo"
-                    [pipeFn]="getPipeFilterFn()"
                     [showChips]="false"
+                    [lazyLoading]="false"
```

Comment:
The committees cannot be lazy loaded, because it will then overwrite the current form, since an incoming committee (from the `getViewModelObservable`-subscription) would update the current form.